management-server/routes/referenceData.js

```diff
@@ -10,28 +10,28 @@ router.get('/', function(req, res, next) {
   res.render('index', { title: 'Veniqa Reference Data' });
 });
 
-router.get('/getCatalogBundle', passportAuth.isAuthenticated, referenceDataController.getCatalogBundle);
```

Comment:
isAuthenticated checks if the user is authenticated. If we remove this, anyone can request it bbasically

Reply:
its being checked in app.js